components/dropdown/dropdown-opener-mixin.js

```diff
@@ -60,10 +60,7 @@ export const DropdownOpenerMixin = superclass => class extends superclass {
 		this._isHovering = false;
 		this._isFading = false;
 
-		this.__onWholeKeypress = this.__onKeypress.bind(this);
```

Comment:
This was leftover from when I was listening at the window level. The only function that actually needs to get bound is the _onOutsideClick function.